packages/main/src/plugin/extension-loader.spec.ts

```diff
@@ -1305,3 +1314,22 @@ test('check listWebviews', async () => {
   expect(result[1].viewType).toBe('anotherView');
   expect(result[1].title).toBe('customTitle2');
 });
+
+test('check version', async () => {
+  const fakeVersion = '1.2.3.4';
+  // mock electron.app.getVersion
+  vi.mocked(app.getVersion).mockReturnValue(fakeVersion);
+
+  const api = extensionLoader.createApi('path', {
+    name: 'name',
+    publisher: 'publisher',
+    version: '1',
+    displayName: 'dname',
+  });
+
+  const readPodmanVersion = api.version;
+
+  // check we called method
+  expect(readPodmanVersion).toBeDefined();
```

Comment:
nit: IMO toBe() call verifies both: to be defined and to be the expected value.

Reply:
@dgolovin amended